static/js/tools/scatter2/info.tsx

```diff
@@ -29,8 +29,8 @@ function Info(): JSX.Element {
       </p>
       <ol>
         <li>
-          Select the type of places you want to plot in the dropdown menu and
-          enter the containing place in the search box above.
+          Enter the containing place in the search box and then select the type
```

Comment:
not sure if the meaning of "containing place" is obvious?

Reply:
changed it to just "enter a place"